STUDIO/org.openl.rules.repository.azure/src/org/openl/rules/repository/azure/AzureCommit.java

```diff
@@ -0,0 +1,71 @@
+package org.openl.rules.repository.azure;
+
+import java.util.Date;
+import java.util.List;
+
+public class AzureCommit {

```

Comment:
package-private access?

Reply:
It's required to be public to be serializable by yaml